functionaltests/src/test/java/specs/scheduler/ScheduledJobFixtureHelper.java

```diff
@@ -58,26 +63,32 @@ public MultiValueResult createPayment(String paymentName,
 
     /**
      * Verify the {@code paymentName} is automatically handled by (short) scheduled task.
+     *
      * @param paymentName payment alias
      * @return map of the payment properties after waiting scheduled task.
      * @throws Exception any exceptions in the tests
      */
     public MultiValueResult verifyPaymentIsHandled(String paymentName) throws Exception {
         Payment payment = paymentFixture.fetchPaymentByLegibleName(paymentName);
 
+        logger.info("Current payment [{}] create at {}, now is {}",
+                payment.getId(), payment.getCreatedAt().toInstant(), ZonedDateTime.now().toInstant());
+
         // wait once, if necessary
         if (payment.getPaymentStatus() == null || StringUtils.isBlank(payment.getPaymentStatus().getInterfaceCode())) {
 
             // how much time to wait till the next scheduled job is definitely started
-            long secondsToWait = payment.getCreatedAt()
-                    .plusSeconds(waitSecondsTimeout())
-                    .minusSeconds(System.currentTimeMillis() / 1000).toEpochSecond();
+            long secondsToWait = waitSecondsTimeout() -
+                    Duration.between(payment.getCreatedAt(), ZonedDateTime.now()).getSeconds();
 
             if (secondsToWait > 0) {
-                LOG.info("Wait up to {} seconds for scheduled job to run", secondsToWait);
+                logger.info("Wait up to {} seconds for scheduled job to run", secondsToWait);
                 Thread.sleep(secondsToWait * 1000);
             }
 
+            logger.info("{} seconds passed after the payment creation - verify the payment now",
```

Comment:
what does it mean `" - verify the payment now"`? just check if it exists? or is there something else to do to verify it?

Reply:
"verify" means assert all the values in this fixture:
https://github.com/commercetools/commercetools-payone-integration/tree/be93c1add566032c3906d86879fc8238f9993ab8/functionaltests/src/test/resources/specs/scheduler

Reply:
Maybe add this info in the log message? because first thing that came to my mind was to check the payment in impex playground

Reply:
no, log messages are only to see them on Travis build and have some "checkpoints" to understand when and why the test is failing.
Actually, all the (automation) tests about verifying/asserting objects states after some actions, doesn't make sense to verify on impex. If in some rare cases this is required - then yes, i'd put a special comment "verify manually on impex playground"

Reply:
ok, maybe I am not familiar with the testing setup of this concordion. But for me, "verify the payment now" doesn't help me where/how to verify, to be honest.

Reply:
it will be done in [`ScheduledJobShort`](https://github.com/commercetools/commercetools-payone-integration/blob/476b3a23d02c64e384dbb6c1f2df3720f6d2aed3/functionaltests/src/test/resources/specs/scheduler/ScheduledJobShort.html)